driver-examples/src/main/java/com/datastax/driver/examples/json/jackson/JacksonJsonColumn.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.driver.examples.json.jackson;
+
+import static com.datastax.oss.driver.api.querybuilder.QueryBuilder.bindMarker;
+import static com.datastax.oss.driver.api.querybuilder.QueryBuilder.insertInto;
+import static com.datastax.oss.driver.api.querybuilder.QueryBuilder.literal;
+import static com.datastax.oss.driver.api.querybuilder.QueryBuilder.selectFrom;
+
+import com.datastax.driver.examples.json.PlainTextJson;
+import com.datastax.driver.examples.json.codecs.JacksonJsonCodec;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.CqlSessionBuilder;
+import com.datastax.oss.driver.api.core.cql.PreparedStatement;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import com.datastax.oss.driver.api.core.cql.Row;
+import com.datastax.oss.driver.api.core.cql.Statement;
+import com.datastax.oss.driver.api.core.type.codec.TypeCodec;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Illustrates how to map a single table column of type {@code VARCHAR}, containing JSON payloads,
+ * into a Java object using the <a href="http://wiki.fasterxml.com/JacksonHome">Jackson</a> library.
+ *
+ * <p>This example makes usage of a custom {@link TypeCodec codec}, {@link JacksonJsonCodec}, which
+ * is implemented in the driver-examples module. If you plan to follow this example, make sure to
+ * include the following Maven dependencies in your project:
+ *
+ * <pre>{@code
+ * <dependency>
+ *     <groupId>com.fasterxml.jackson.core</groupId>
+ *     <artifactId>jackson-databind</artifactId>
+ *     <version>${jackson.version}</version>
```

Comment:
I don't think this will get resolved by Maven, I suggest that you use a raw version number, if possible the latest version available.